gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# 143 squads back end

### Description

1. added squad models
2. added squad swipe model

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users

1. open python shell
2. check if every user has a squad associated with it
3. check if every squadswipe is empty initially
4. implement front end and test backend for squadswipe

```

### Miscellaneous Comments

### Checklist
